web/src/webpack.prod.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = (env) => ({
     filename: 'static/[name].js',
   },
   resolve: {
-    extensions: ['.ts', '.tsx', '.js', '.jsx', '.css'],
+    extensions: ['.mjs', '.ts', '.tsx', '.js', '.jsx', '.css'],
```

Comment:
I'm not familiar with this extension `.mjs`, could you help explain why this is needed to upgrade Apollo?